src/security-council-mgmt/governors/modules/SecurityCouncilNomineeElectionGovernorCountingUpgradeable.sol

```diff
@@ -84,34 +98,39 @@ abstract contract SecurityCouncilNomineeElectionGovernorCountingUpgradeable is
             "SecurityCouncilNomineeElectionGovernorCountingUpgradeable: Not enough tokens to cast this vote"
         );
 
-        uint256 prevVotesReceived = election.votesReceived[candidate];
+        uint256 prevVotesReceived = election.votesReceived[contender];
         uint256 votesThreshold = quorum(proposalSnapshot(proposalId));
 
-        emit VoteCastForContender(proposalId, account, candidate, votes);
-
-        if (prevVotesReceived + votes < votesThreshold) {
-            // we didn't push the candidate over the line, so just add the votes
-            election.votesUsed[account] = prevVotesUsed + votes;
-            election.votesReceived[candidate] = prevVotesReceived + votes;
-        } else {
-            // we pushed the candidate over the line
-            // we should only give the candidate enough votes to get to the line so that we don't waste votes
-            uint256 votesNeeded = votesThreshold - prevVotesReceived;
+        uint256 actualVotes = votes;
 
-            election.votesUsed[account] = prevVotesUsed + votesNeeded;
-            election.votesReceived[candidate] = prevVotesReceived + votesNeeded;
+        if (prevVotesReceived + votes >= votesThreshold) {
+            // we pushed the contender over the line
+            // we should only give the contender enough votes to get to the line so that we don't waste votes
+            actualVotes = votesThreshold - prevVotesReceived;
 
-            // push the candidate to the nominees
-            election.nominees.push(candidate);
+            // push the contender to the nominees
+            election.nominees.push(contender);
 
-            emit NewNominee(proposalId, candidate);
+            emit NewNominee(proposalId, contender);
         }
+
+        election.votesUsed[account] = prevVotesUsed + actualVotes;
+        election.votesReceived[contender] = prevVotesReceived + actualVotes;
+
+        emit VoteCastForContender({
+            proposalId: proposalId,
+            voter: account,
+            contender: contender,
+            votes: actualVotes,
+            totalUsedVotes: prevVotesUsed + actualVotes,
+            totalUsableVotes: weight
```

Comment:
Minort thing, but maybe this should just be `usableVotes` rather than `totalUsableVotes`